Cargo.toml

```diff
@@ -44,10 +55,6 @@ gfx_window_sdl = { path = "src/window/sdl", version = "0.4" }
 gfx_device_dx11 = { path = "src/backend/dx11", version = "0.4" }
```

Comment:
I wonder if we should feature gate DX11 the same way for consistency
the fact that DX11 is used by default doesn't help the users too much anyway